analysis/post_release/manuscript_table1.R

```diff
@@ -9,15 +9,15 @@ print("Pivot table")
 
 df <- tidyr::pivot_wider(df, 
                          names_from = "cohort",
-                         values_from = c("N (%)","COVID-19 diagnoses"))
+                         values_from = c("N (%) midpoint6 derived","COVID-19 diagnoses midpoint6"))
 
 # Tidy table -------------------------------------------------------------------
 print("Tidy table")
 
 df <- df[,c("Characteristic","Subcharacteristic",
-            paste0(c("N (%)","COVID-19 diagnoses"),"_prevax"),
-            paste0(c("N (%)","COVID-19 diagnoses"),"_vax"),
-            paste0(c("N (%)","COVID-19 diagnoses"),"_unvax"))]
+            paste0(c("N (%) midpoint6 derived","COVID-19 diagnoses midpoint6"),"_prevax"),
+            paste0(c("N (%) midpoint6 derived","COVID-19 diagnoses midpoint6"),"_vax"),
+            paste0(c("N (%) midpoint6 derived","COVID-19 diagnoses midpoint6"),"_unvax"))]
```

Comment:
[Optional] You might want to rename these columns to remove `midpoint6` and `midpoint6 derived` because this will likely be added as a footnote in the results pack (instead of being in every column name).

Reply:
Made the change by adding an extra step.